jsapp/scss/components/_kobo.drawer.lists.scss

```diff
@@ -85,7 +84,7 @@
   &__item {
     display: block;
     position: relative;
-    padding-left: 30px;
+    padding-left: 13px;
```

Comment:
I think you can go with a smaller number here - `3px` will align items with group header icons